packages/nimble-components/src/mapping/base/tests/story-helpers.ts

```diff
@@ -0,0 +1,3 @@
+export function keyDescription(renderedThing: string): string {
```

Comment:
I presume this is being exported here with the expectation that there will be a separate stories file that will leverage the same description?

Reply:
Yep, since I already have both columns implemented, but I'm splitting up the changes, you may see stuff like this or seemingly unnecessary base classes, etc.